gooddata-sdk/gooddata_sdk/visualization.py

```diff
@@ -405,6 +460,54 @@ def __repr__(self) -> str:
         return repr(self._filter)
 
 
+class VisualizationSortLocator:
+    def __init__(self, locator: dict[str, dict[str, str]]) -> None:
+        locator_key = list(locator.keys())[0]
```

Comment:
It is not safe to pass the dictionary here and take the first value of keys. The order in a dictionary can change. I suggest passing the locator and the type directly. There is no need to pass the whole dictionary. It is not stored anywhere, only accessed.